tests/aws/services/stepfunctions/v2/scenarios/test_base_scenarios.py

```diff
@@ -1121,6 +1121,35 @@ def test_map_state_tolerated_failure_percentage_path(
             exec_input,
         )
 
+    @markers.aws.validated
```

Comment:
Great, let's simplify this test case by avoiding s3 setups and have the map run execute simple `pass` states.

Reply:
Fixed in [c1129ef](https://github.com/localstack/localstack/pull/11073/commits/c1129efb638c6b82d40296a996560dd681a46ab0).